spec/factories/fake_org_nummers.rb

```diff
@@ -0,0 +1,45 @@
+# Generate a valid Swedish OrganisationNummer (OrgNummer; OrgNr).
+#
+# It must be 10 digits and the last digit must be the valid Luhn checksum, per
+# https://sv.wikipedia.org/wiki/Organisationsnummer
+#
+
+require_relative '../../app/services/luhn_checksum'
+
+class FakeOrgNummers
+
+  MAX_NUM = 9999999999 unless defined?(MAX_NUM)
+  MIN_NUM = 1 unless defined?(MIN_NUM)
```

Comment:
Shouldn't this be `0000000000`?

Reply:
It could be `0`-- I sort of arbitrarily picked `1`.  (The final result will be 0-padded so it's 10 digits.  Not sure if you were alluding that that at all.)

Reply:
No, actually I was alluding to having a range of random numbers produce that could include numbers with less than 10 digits.  So, to correct my prior comment, perhaps a more efficient way is to set `MIN_NUM` to `1000000000`.   

That way we'll only generate numbers with 10 digits.

Reply:
You will get a 10 digit string of the form /\d\d\d\d\d\d\d\d\d\d/
but it may be 0 padded.
IOW,  '0000000000' is a valid Company number (for testing, anyway; the `orgnummer gem` will say that it is valid, which is what we care about for testing)

I can't think of a reason to limit it to just [1000000000..9999999999]  but we certainly could.

Reply:
Not a big deal, since most of the time the random number generator will return a 10-digit number.  But, with a floor lower than `1000000000` we'll get some numbers with less digits:
```
 r.rand(0..9999999999)
 => 2054828604 
2.4.2 :020 > r.rand(0..9999999999)
 => 8894396888 
2.4.2 :021 > r.rand(0..9999999999)
 => 2436793130 
2.4.2 :022 > r.rand(0..9999999999)
 => 123617672 
```

Reply:
The `company_number` is a String (varchar in the db)

And so a string is always what is generated.  It is always 10 digits long.  I use  `"%010d"` to format it.

```
>> num = Random.rand(0..9999999999)
=> 279535843
>> "%010d" % num
=> "0279535843"
```

So with your example, the actual _company_number_ returned by the Generator is a String: "0123617672"
According to the `org_nummer` gem,  "0000000000" is a perfectly valid company number.

Am I missing something?  I'm not seeing what the problem is with using (0..9999999999)